lib/shuffle.js

```diff
@@ -115,8 +115,8 @@ module.exports = function shuffle(array) {
     for (let i = array.length - 1; i > 0; i--) {
         const randIndex = randomRange(0, i);
         const randIndexVal = array[randIndex];
-        array[randIndex] = array[i];
-        array[i] = randIndexVal;
+        array[randIndex] = array[i]; // eslint-disable-line no-param-reassign
```

Comment:
This is sad, I'll try to get rid of it later, since we're using the returned array.